src/status_im2/contexts/onboarding/create_password/style.cljs

```diff
@@ -1,14 +1,66 @@
 (ns status-im2.contexts.onboarding.create-password.style
-  (:require [quo2.foundations.colors :as colors]
-            [react-native.platform :as platform]))
+  (:require [quo2.foundations.colors :as colors]))
 
-(def navigation-bar {:height 56})
+(def image-background
+  {:height "100%"
+   :width  "100%"})
 
-(def page-container
-  {:padding-top      (if platform/ios? 44 0)
-   :position         :absolute
+(def overlay
+  {:position         :absolute
    :top              0
    :bottom           0
    :left             0
    :right            0
    :background-color colors/neutral-80-opa-80-blur})
+
+(def content-style {:flex-grow 1})
+
+(def heading {:margin-bottom 20})
+(def heading-subtitle {:color colors/white})
+(def heading-title (assoc heading-subtitle :margin-bottom 8))
+
+(def label-container
+  {:margin-top     8
+   :flex-direction :row
+   :height         16})
+
+(def label-icon
+  {:width        16
+   :height       18
+   :margin-right 4})
+
+(defn label-icon-color
+  [status]
+  (status {:neutral colors/neutral-40
```

Comment:
this looks strange, what is status and why we pass it as parameter?

Reply:
`status` is a keyword indicating what is the current state of the label below the inputs, it can be `:neutral`, `:success` or `:danger`.  this function returns the icon color in the label according to its style.

I modified the code to look as follows:
```clojure
(defn label-icon-color
  [status]
  (get {:neutral colors/neutral-40
        :success colors/success-60
        :danger  colors/danger-60}
       status))
```

I hope this helps to understand we are extracting the status from the map :)